### PR TITLE
another commit

### PR DESCRIPTION
**Pull request by advaitds**
Added another comment on Schroedinger.

### CODE REVIEW

File: somefile.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
 hello Advait.
 how are you?
-i think that schroedinger is a bit dull. 
\ No newline at end of file
+i think that schroedinger is a bit dull. 
+
+Hello Galib.
+I am .. existing. 
+I agree that schoedinger is quite dull. 
+But you know he solved the hydrogen atom.
```
Comment (galibhassan) [inline somefile.txt:8]: Hmm. You have a point. But try giving him the _Uranium atom_. 

So could you please make some changes in the word `dull`?

## Todo
Just add three or four more `L`s at the end of `dull` and then commit and execute:
```
git push
```
Your PR will be automatically updated. And I will be happy to merge it. :neckbeard:

Comment (advaitds) [reply somefile.txt:8]: Thanks for the feedback :)
Schroedinger is successfully more "dulllled".

Comment (galibhassan) [reply somefile.txt:8]: Cool! Now you can merge the pull request.
image

choosing `squash and merge` will squash all the commits into one, and then place that commit on top of the latest commit in the commit history. So, do it! Commit message can also be changed during this process.

Comment (advaitds) [reply somefile.txt:8]: "This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests..."

So I guess you have to do it.. or give me access rights.

Comment (galibhassan) [reply somefile.txt:8]: Cool! Merging this.

**Pull request merged**